docs/resources/windows_firewall_rule.md.erb

```diff
@@ -0,0 +1,137 @@
+---
+title: About the windows_firewall_rule Resource
+platform: windows
+---
+
+# windows_firewall_rule
+
+Use the `windows_firewall_rule` Chef InSpec audit resource to test if a firewall rule is correctly configured on a Windows system.
+
+<br>
+
+## Availability
+
+### Installation
+
+This resource is distributed along with Chef InSpec itself. You can use it automatically.
+
+## Syntax
+
+A `windows_firewall_rule` resource block specifies which rule to validate:
+
+    describe windows_firewall_rule('name') do
+      it { should be_enabled }
+    end
+
+where
+
+* `('name')` must specify the name of a firewall rule (not the display name)
+* `be_enabled` is a valid matcher for this resource
+
+<br>
+
+## Examples
+
+The following examples show how to use this Chef InSpec audit resource.
+
+### Test if the firewall contains a rule for outbound HTTPS
+
+    describe windows_firewall_rule('HTTPS Out') do
+      it { should be_enabled }
+      it { should be_allowed }
+      it { should be_outbound }
+      it { should be_tcp }
+
+      its('remote_port') { should eq 443 }
+    end
+
+<br>
+
+## Properties
+
+The resource compiles the following list of firewall rule properties:
+
+* `description`
+* `displayname`
+* `group`
+* `local_address`
+* `local_port`
+* `remote_address`
+* `remote_port`
+* `direction`
+* `protocol`
+* `icmp_type`
+* `action`
+* `profile`
+* `program`
+* `service`
+* `interface_type`
+
+Each of these properties can be used in two distinct ways:
```

Comment:
Should there be an "or" inserted between line 72 and line 73 to emphasize what are the two distinct formats/ways?